source/sonata_simulation.rst

```diff
@@ -96,6 +97,27 @@ example::
        "integration_method" : 2
   }
 
+Format of the electrodes_file
```

Comment:
I think the file format could be added to the `sonata_tech.rst` file, instead of `SONATA Simulation Configuration file` page

Reply:
Is the intention of the `sonata_tech.rst` file to describe all of the input file formats? Since the electrodes file seems to me to be quite different from the node and edge files, I think it might be better to put the documentation in a dedicated page. That said, I can put it in `sonata_tech.rst` if that's the consensus opinion

Reply:
`sonata_tech.rst` covers "The following is the definition of the file format used in the SONATA file.", so I think it's an appropriate place.